src/TLuaInterpreter.cpp

```diff
@@ -12517,6 +12521,14 @@ void TLuaInterpreter::initIndenterGlobals()
     luaL_dostring(pIndenterState, QStringLiteral("package.cpath = package.cpath .. ';%1/lib/?.so'")
                   .arg(QCoreApplication::applicationDirPath())
                   .toUtf8().constData());
+#elif defined(Q_OS_WIN32)
+    // For Qt Creator builds, add search paths one and two levels up from here, then a 3rdparty directory:
+    luaL_dostring(pIndenterState, QStringLiteral("package.path = [[" LUA_DEFAULT_PATH R"(\?.lua;%1\..\3rdparty\?.lua;]] .. package.path)")
+                  .arg(QDir::toNativeSeparators(QCoreApplication::applicationDirPath()))
+                  .toUtf8().constData());
+    luaL_dostring(pIndenterState, QStringLiteral(R"(package.path = [[%1\..\..\3rdparty\?.lua;]] .. package.path)")
```

Comment:
This should be included in the previous `luaL_dostring(`...`)` you can use the `%1` as many times as you need it so these would all condense down to become:

    +    luaL_dostring(pIndenterState, QStringLiteral("package.path = [[" LUA_DEFAULT_PATH "\?.lua;%1\..\3rdparty\?.lua;%1\..\..\3rdparty\?.lua;]] .. package.path")
    +                  .arg(QDir::toNativeSeparators(QCoreApplication::applicationDirPath()))
    +                  .toUtf8().constData());

Note that I have dropped the C++11 raw string literal because it is not really helping here...

Reply:
The raw string helps, because without it, I'd need to escape the backslash, no?

Reply:
:man_facepalming: duh, yeah... I'd missed that, what with the preprocessor insertion of the LUA_DEFAULT_PATH into the two other strings and the lua `[[`...`]]` string wrapping.  I suppose as a `QStringLiteral(`...`)` composed at compilation time it is possibly acceptable to move the `LUA_DEFAULT_PATH` as an `QString::arg()` so that a single C++11 string literal could be used for the whole rather than just a part of the string.  Of course, until [QTBUG-42736](https://bugreports.qt.io/browse/QTBUG-42736) is resolved we will probably have to abandon _any_ C++11 string literals in any _compilation unit_ (which I understand to mean any C++ class `.cpp` file) that contains any `QObject::tr(`...`)` or related calls... :sob:

Reply:
All right, I'll have a look